pages/search.tsx

```diff
@@ -237,7 +238,7 @@ class Search extends Component<Props, State> {
           {
             (!isSearching && searchCompleted && listItemNodes && listItemNodes.length === 0) &&
               <div className='no-results-msg'>
-                No podcasts {`${searchBy === 'host' ? 'with that host' : ''}`} found.
+                No podcasts {`${searchBy === PV.query.host ? 'with that host' : ''}`} found.
```

Comment:
'with that host' is also a string we'll need to translate.

I should rewrite this no results message though. It'll be better as a function that takes 2 parameters, search type ("podcasts") and altText ("with that host"), and returns this string. I'll search for the className no-results-msg to find other parts of the code using a similar message...